articles/continuous-system-integration.md

```diff
@@ -0,0 +1,41 @@
+
+Continuous Integration on the System Level
+
+System CI helps with avoiding such “big bang integration” effects. It is far from realistic to achieve the continuity of the fast integration cycles of software, e.g., check-in builds, on the system level. However, the principle of continuously integrating parts to a whole applies at the system level, too.
+The continuity of software integration is based on the integration of source code parts into the mainline whereas the continuity of system integration is based on the availability of system parts over time, e.g., PCBs, mechanical parts, cables, software functionality, infrastructure, test stands, etc.
+
+The evolution of the System CI is driven by a roughly planned “integration vision”. This may start as just a sketch on a whiteboard depicting how the successively available system parts might be put together over time. The integration vision is not a detailed plan, but a guidance for the integration activities on the system level.
+The concrete integration of parts is planned and performed just as they become available – or maybe a short time ahead. The integration itself is performed as simply as possible. For example, a specially developed sophisticated mechanical fixation is not needed if a tape is sufficient to attach one part to another in an early lab model.
+
+System CI is a constant flow of adding parts and replacing parts with, for example, more matured parts or new revisions of parts. The evolving system is the central target of the System CI infrastructure. The infrastructure takes care of, among other things, building the software, deploying it to the integrated system, and automatically testing the realized system functionality.
+This allows for regression testing on the system level, which is important in dealing with all the changes on the system integration – as software regression testing is important in dealing with all the changes in software development.
+
+Fail Early!
+
+System CI enables us to fail early and thus gives us time to react, change, and test again in fast cycles. Hence, System CI heavily reduces the risk of late design breakage. Additionally, the progress of system development does not have to be deduced or guessed indirectly, but can be demonstrated with the current state of the integrated system – at any point of time along the development lifecycle.
+
+The order of particularly meaningful system integration steps needs to be planned, at least roughly. Given a product vision and a draft system architecture, a value- and risk-based system backlog planning are the basis for a successful continuous system integration that lowers the risk of late design breakage and increases the opportunity to develop the right product.
```

Comment:
Die Informationen dieses Abschnittes würde ich im Abschnitt auf Zeile 28 unterbringen (der Abschnitt welcher mit "System CI is a constant flow …" 

Der obige Abschnitt mit der Aufzählung der Vorteile ist an sich ein guter Schluss. 

Allenfalls kannst du den Artikel mit einer Handlungsaufforderung abschliessen: Give it a try! oder "Increase the opportunity to develop the right product by applying this practice in your next systems project!

Reply:
Auch für diese Hinweise bin ich Dir sehr dankbar. Die Qualität des Endes wird damit deutlich besser als vorher. Wie oben schon angemerkt, ich bin mehr der Typ für Ausrufezeichen und habe deswegen Deinen Vorschlag für das Ende genau so übernommen. Also nochmals vielen Dank für das kritische und hilfreiche Review und die Zeit die Du dafür investierst!!!